modules/imap/site.js

```diff
@@ -1313,19 +1313,78 @@ var add_email_in_contact_trusted = function(list_email) {
     }
 };
 
+var list_blocked_senders = [];
+get_list_block_sieve(Hm_Utils.parse_folder_path(hm_list_path()));
```

Comment:
I believe recent master has a dispatch/routing module that has other ways to execute code on page load and not on file load like you did it here. If in doubt, ask @mercihabam about it.